### DISCUSSION TITLE
Can i change the size of the icon?

### DISCUSSION THREAD

**Discussion by MathiewMay2** [requarks/wiki#2964] [General]
Hi, i would like to change the size of the icon you see in the top left of the page.

I tried using css override but it did not change anything

.nav-header-inner .v-avatar{
height: 55px;
min-width: 55px;
}

this is what i set in css override.

**Comment by NGPixel** [comment]
It cannot be changed using CSS unfortunately. Your only option would be to fork the source code, change this line and re-compile it:

https://github.com/Requarks/wiki/blob/2de0a74c8ed7de1bcd9d41b6ea2566e710b0467d/client/components/common/nav-header.vue#L23
